test/e2e/template/deployment-rbac.yaml

```diff
@@ -41,6 +41,19 @@ spec:
     plural: azureidentities
   scope: Namespaced
 ---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: azurepodidentityexceptions.aadpodidentity.k8s.io
```

Comment:
We should add it in the deploy file under master directory as welll. And then move that to the deployment file in the top level folder.